appcues/src/debug/java/com/appcues/CompositionTestHelper.kt

```diff
@@ -35,6 +47,65 @@ fun ComposeContent(json: String, imageLoader: ImageLoader) {
     }
 }
 
+@Composable
+fun ComposeContainer(
+    context: Context,
+    experienceJson: String,
+    traitJson: String,
```

Comment:
considering having this take a list of trait JSON to apply, but I think for now it's not needed and we can always expand that later